pkg/snowflake/user.go

```diff
@@ -62,6 +65,17 @@ func ScanUserDescription(rows *sqlx.Rows) (*user, error) {
 			r.DefaultNamespace = userProp.Value
 		case "DEFAULT_ROLE":
 			r.DefaultRole = userProp.Value
+		case "DEFAULT_SECONDARY_ROLES":
```

Comment:
i am a bit confused what is happening here. why unmarshal it to a []string only to convert it back to a sql string? do you have a link to any documentation on this?

Reply:
@sfc-gh-swinkler you're right it wasn't necessary - I refactored it.